src/rest/Facet/FacetSearchResponse.ts

```diff
@@ -5,6 +5,7 @@ export interface IFacetSearchResultValue {
    * **Example:** `ACME Product A`
    */
   displayValue: string;
+  path: string[];
```

Comment:
The [Swagger documentation](https://platformdev.cloud.coveo.com/docs/?api=SearchApi#!/Search/post_rest_search_v2_facet)'s model didn't contain a description for this attribute. It defines the parent path (not full path) to the value.